### PR TITLE
Added Bot Sharder

### PR DESCRIPTION
**Pull request by TheMonDon**
**Please describe the changes this PR makes and why it should be merged:**
Added custom-files directory with Bot Sharder from custom-files repo

**Status**

- [X] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [X] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: custom-files/Bot Sharder/sharded-bot.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    'This version of Discord Bot Maker requires Discord.JS v12.\nPlease use "Project > Module Manager" and "Project > Reinstall Node Modules" to update to Discord.JS v12.',
+  );
+  throw new Error('Need Discord.JS v12 to Run!!!');
+}
+
+console.log('-'.repeat(50));
+console.log("TheMonDon's DBM Bot Sharder");
+console.log(`Version: ${version}`);
+console.log("You can change the amount of shards by providing '--shard_count=[number]' (default: auto)");
+console.log("You can change the bot file by providing '--startup=./index.js' (default bot.js)");
```
Comment (tlfn) [inline custom-files/Bot Sharder/sharded-bot.js:18]: Why do you put index.js instead of placeholders like [file]

Comment (TheMonDon) [reply custom-files/Bot Sharder/sharded-bot.js:18]: Just to show that it requires './[file]', though could've done that. Not much of a difference.

**Review by almeidx — CHANGES_REQUESTED** [review]
Instead of a custom-files folder, create a "sharder" folder on the root and put the file in it instead

File: sharder/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+The default `shard_count` flag is set to `auto` (determined by discord.js), which is fine for most bots.
+
+To change the bot startup file, add `--startup=./index.js` after `node sharded-bot.js`; ex. `node sharded-bot.js --startup=./index.js`.
+The default `startup` parameter is set to `bot.js`.
+
+**If you want to do anything across shards you will need to use** [`client.shard.broadcastEval()`]
+
+For more information in regards to sharding, check [this guide].
+
+[zip file]: https://downgit.github.io/#/home?url=https://github.com/dbm-network/custom-files/blob/master/Bot%20Sharder/sharded-bot.js
```
Comment (almeidx) [inline sharder/README.md:31]: This URL needs to be updated

File: sharder/sharded-bot.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  }
+} else {
+  console.error('Could not find the settings.json file');
+}
+
+if (!token) {
+  console.error("Token must be supplied in 'settings.json' in the data folder, double check your bot settings!");
+}
+
+// Create your ShardingManger instance
```
Comment (almeidx) [inline sharder/sharded-bot.js:90]:
```suggestion
```

**Review by almeidx — CHANGES_REQUESTED** [review]

File: sharder/README.md (hunk 2 above)
Comment (almeidx) [reply sharder/README.md:31]: Also, use https://dbm-network.github.io/download-git instead

**Comment by TheMonDon** [comment]
Hopefully, the URL will be correct. Can't test without the file actually being there.

**Review by almeidx — APPROVED** [review]

**Pull request merged**